_summaries/brown2020language_1.md

```diff
@@ -1,19 +1,90 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Language Models are Few-Shot Learners
+paper: brown2020language
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: jaywhang
+score: 10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea
+
+Builds upon their previous work [cite], this paper introduces GPT-3, a 175B-parameter autoregressive language model.
+The main discovery of this paper is the model's ability to perform few-shot learning **without gradient steps or fine-tuning**.
+This paper presents an extremely thorough large-scale study of GPT-3's performance on a wide range of NLP tasks.
+Their experiments also test GPT-3 on a few non-standard tasks that further push the limits of of the pre-trained language model.
+
+## Technical Description
+
+### Training Details
+
+GPT-3 largely follows the architecture of GPT-2, with a minor difference of having alternating dense and sparse (locally-banded) attention.
+In order to train such a large model, the authors also created a custom text corpus.
+Specifically, they combined five different datasets with custom preprocessing to ensure the quality of examples.
+The exact weight and size the number of tokens (after Byte-Pair Encoding) is shown in **Table 1**.
+
+[Table 1: Include Table 2.2 from the paper]
+
+Due to the size of their model, they determined the learning rate and batch size based on the level of gradient noise.
+Lastly, to accomodate such a large model with extremely large batch size (~3.2M), they employed model parallelism across V100 GPUs.
+
+### Evaluation Details
+
+To test the model's few-shot learning capability, they sampled examples from the training data for each task and fed that as conditioning input, followed by 1-2 blank lines.
```

Comment:
You can add some details on how few-shot learning is set up, e.g. zero-shot, one-shot, few-shot (Figure 2.1).

Reply:
Thanks for the comment.  I have added a description of their task formulation . I also improved the existing text and completed all the missing parts.